lib/ecto/query/api.ex

```diff
@@ -478,6 +478,35 @@ defmodule Ecto.Query.API do
   """
   def fragment(fragments), do: doc! [fragments]
 
+  @doc """
```

Comment:
```suggestion
  @doc ~S'''
```

This way you don't need to escape `"""` in the examples inside nor interpolations.